dawn/renderer/components/Editor.js

```diff
@@ -119,6 +120,10 @@ class Editor extends React.Component {
     }, { passive: true });
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    return nextProps !== this.props || nextState !== this.state;
```

Comment:
I can try to check if this switch causes better improvements later, during state revision.